app/ui-react/packages/api/src/WithVirtualizationHelpers.tsx

```diff
@@ -111,6 +122,49 @@ export class WithVirtualizationHelpersWrapped extends React.Component<
     return Promise.resolve();
   }
 
+  /**
+   * Unpublish the Service VDB with the specified name.
```

Comment:
minor thing here - comment isn't applicable to the function

Reply:
see below